tensorflow/python/keras/layers/normalization.py

```diff
@@ -922,6 +926,7 @@ def __init__(self,
                beta_constraint=None,
                gamma_constraint=None,
                trainable=True,
+               fused=False,
```

Comment:
Please put "fused" after "name". New arguments must always be added to the end of the argument list.